wireless_modules/middle_module/README.md

```diff
@@ -0,0 +1,62 @@
+# DAS - MIDDLE WIRELESS MODULE
+
+[![All Contributors](https://img.shields.io/badge/all_contributors-6-orange.svg?style=flat-square)](#contributors)
+
+The Middle Wireless Module allows us to collect and send data from the sensors attached to the bike. The middle wireless module consists of a micro controller (ESP32 currently) which gathers the data from the on board sensors and sends them using MQTT (over the internet).
+
+This repository contains all the code related to Monash Human Power's middle wireless module - part of their Data Acquisition System (DAS).
+
+## Getting Started
+
+1. Press clone on the root folder and use it to clone the entire repository on the command line.
+2. Create a ``config.py`` file in the same directory as the contents of this folder (refer below).
+3. The ``config.py`` file should hold the following variables (ensure the name matches):
+- ``ESSID`` : Should be assigned to a string containing the SSID of the network to connect to
+- ``PASSWORD``: Should be assigned to a string containing password of the network
+- ``MQTT_BROKER``: A string containing either the web server name or IP address which acts like the Mqtt broker
+- ``builtin_LED = PIN(2, PIN.OUT)``
+
+## Contents
+- ``boot.py``: Contains function to connect the micro controller to the internet
+- ``main.py``: Processes MQTT messages and sends sensor data
+- ``MQTT_Client_class.py``: Holds the Client class (used to work with MQTT)
+- ``test.py``: A python file (NOT Micropython) used to run on the local computer to test the ``main.py`` and ``MQTT_Client_class.py`` files. It basically sends on the ``/v3/wireless-module/<-id->/start`` topic, to check whether it initiates the ``main.py`` file to send through the data on the topic ``/v3/wireless-module/<-id->/data`` and then send on the topic ``/v3/wireless-module/<-id->/stop`` to check whether it stops the inflow data. 
+	- Note: There's some latency with MQTT and so after the stop topic is used the ``test.py`` finishes running and so may not print the last few data sent by ``main.py``.
+	- Also when running ``test.py`` if you do not receive any messages, re-run because most likely ``main.py`` was not ready to read the message from the start topic when it was sent by ``test.py`` (This may be due to some error and the need for the ESP32 to restart which would take some time)
+
+
+## Dependencies
+- The ``test.py`` requires the ``paho-mqtt`` library installed for Python
+- The other ``.py`` files are Micropython files so needs a board to run on (eg ESP32)
+- Ensure the ``config.py`` file is created locally (see the 'Getting Started' section above)
+
+
+## Steps to test?
+1. Download all the files (clone the root repository)
+2. Write a ``config.py`` file with the relevant variables (see above)
+3. Read this page (especially ensuring you understand what ``test.py`` does and what dependencies exist)
```

Comment:
This duplicates the Getting Started section

Reply:
agreed I'll remove it